test/scripts/agent-images/Containerfile-e2e-base.local

```diff
@@ -8,7 +8,10 @@
 # * flightctl-agent configured to talk to our e2e kind flightctl-server
 # * greenboot and podman-compose installed
 
-FROM quay.io/centos-bootc/centos-bootc:stream9
+FROM quay.io/fedora/fedora-bootc:41
```

Comment:
I know you need this for testing but we need to drop that here :)

Reply:
The PR e2e tests didn't pass without it.  I can try again.

Reply:
Removed